gulpfile.js

```diff
@@ -786,6 +787,49 @@ gulp.task(
   })
 );
 
+// Builds the generic production viewer that should be compatible with most
+// older HTML5 browsers.
+gulp.task(
+  "generic-es5",
+  gulp.series("buildnumber", "default_preferences", "locale", function() {
+    console.log();
+    console.log("### Creating generic (ES5) viewer");
+    var defines = builder.merge(DEFINES, { GENERIC: true, SKIP_BABEL: false });
+
+    rimraf.sync(GENERIC_ES5_DIR);
+
+    return merge([
+      createBundle(defines).pipe(gulp.dest(GENERIC_ES5_DIR + "build")),
+      createWebBundle(defines).pipe(gulp.dest(GENERIC_ES5_DIR + "web")),
+      gulp
+        .src(COMMON_WEB_FILES, { base: "web/" })
+        .pipe(gulp.dest(GENERIC_ES5_DIR + "web")),
+      gulp.src("LICENSE").pipe(gulp.dest(GENERIC_ES5_DIR)),
+      gulp
+        .src(
+          ["web/locale/*/viewer.properties", "web/locale/locale.properties"],
+          { base: "web/" }
+        )
+        .pipe(gulp.dest(GENERIC_ES5_DIR + "web")),
+      gulp
+        .src(["external/bcmaps/*.bcmap", "external/bcmaps/LICENSE"], {
+          base: "external/bcmaps",
+        })
+        .pipe(gulp.dest(GENERIC_ES5_DIR + "web/cmaps")),
+      preprocessHTML("web/viewer.html", defines).pipe(
+        gulp.dest(GENERIC_ES5_DIR + "web")
+      ),
+      preprocessCSS("web/viewer.css", "generic", defines, true)
+        .pipe(postcss([autoprefixer(AUTOPREFIXER_CONFIG)]))
```

Comment:
This appears to be missing the CSS variables/`calc` plugins.

Reply:
Yes, more rebase errors.
(We may want to look into *not* providing polyfilled CSS for the modern builds, but since those things aren't as easy to polyfill as missing JS features I decided to leave that alone for now.)